mesheryctl/internal/cli/root/app/view.go

```diff
@@ -2,21 +2,36 @@ package app
 
 import (
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"regexp"
 	"strings"
+	"time"
 
 	"github.com/ghodss/yaml"
+	"github.com/gofrs/uuid"
+	"github.com/layer5io/meshery/internal/sql"
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
 	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/constants"
 	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/layer5io/meshery/models"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+type appStruct struct {
+	Name            string
+	ID              *uuid.UUID
+	ApplicationFile string
+	UserID          *string
+	Location        sql.Map
+	UpdatedAt       *time.Time
+	CreatedAt       *time.Time
+}
+
```

Comment:
we have this in models.MesheryApplication so we can use that here.

Reply:
good one. Resolved now